source/exe/service_main_entry.cc

```diff
@@ -0,0 +1,30 @@
+#include "nighthawk/common/exception.h"
+
+#include "client/service_main.h"
+
+#include "absl/debugging/symbolize.h"
+
+// NOLINT(namespace-nighthawk)
+
+using namespace Nighthawk::Client;
+
+int main(int argc, const char** argv) {
+#ifndef __APPLE__
```

Comment:
@jmarantz, this looks like something you might be knowledgeable about as a reviewer...

Reply:
this seems OK. Probably we should wrap this in a helper function to hide the ifdef, but not critical as it happens only once in this repo.